docs/api/metrics.md

```diff
@@ -9,23 +9,37 @@ The External Secrets Operator exposes its Prometheus metrics in the `/metrics` p
+|------------------------------------------------|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `externalsecret_provider_api_calls_count`      | Counter   | Number of API calls made to an upstream secret provider API. The metric provides a `provider`, `call` and `status` labels.                                                                                              |
+| `externalsecret_sync_calls_total`              | Counter   | Total number of the External Secret sync calls                                                                                                                                                                          |
+| `externalsecret_sync_calls_error`              | Counter   | Total number of the External Secret sync errors                                                                                                                                                                         |
+| `externalsecret_status_condition`              | Gauge     | The status condition of a specific External Secret                                                                                                                                                                      |
+| `externalsecret_reconcile_duration`            | Gauge     | The duration time to reconcile the External Secret                                                                                                                                                                      |
+
+## Cluster Secret Store Metrics
+| Name                                    | Type  | Description                                             |
+|-----------------------------------------|-------|---------------------------------------------------------|
+| `clustersecretstore_status_condition`   | Gauge | The status condition of a specific Cluster Secret Store |
+| `clustersecretstore_reconcile_duration` | Gauge | The duration time to reconcile the Cluster Secret Store |
+
+# Secret Store Metrics
+| Name                             | Type  | Description                                     |
+|----------------------------------|-------|-------------------------------------------------|
+| `secretstore_status_condition`   | Gauge | The status condition of a specific Secret Store |
+| `secretstore_reconcile_duration` | Gauge | The duration time to reconcile the Secret Store |
+
+## Controller Runtime Metrics
+See [the kubebuilder documentation](https://book.kubebuilder.io/reference/metrics-reference.html) on the default exported metrics by controller-runtime.
```

Comment:
The original list does not seem to include all the controller runtime metrics, and since kubebuilder already has dedicated page for the metrics, I thought it might be better if we simply link to it 🙂